lib/medium-admin.php

```diff
@@ -362,6 +489,29 @@ private static function _get_boolean_options() {
     );
   }
 
+  /**
+   * Returns an array of the publications the supplied user can publish into.
+   */
+  private static function _get_user_publication_options(Medium_User $medium_user) {
+    $default_option = new stdClass();
+    $default_option->name = __("None", "medium");
+    $default_option->publishable = true;
+    $options[NO_PUBLICATION] = $default_option;
```

Comment:
does this line automagically initialize $options as an array? (I don't see $options being defined anywhere)